### ISSUE TITLE
Hyphens in stoa0255.stoa010.perseus-lat2.xml

### ISSUE THREAD

**Issue by nevenjovanovic** [PerseusDL/canonical-latinLit#90]
Latin text in Seneca's De ira sometimes keeps unmarked (unencoded) hyphens at the end of lines, sometimes even with an extra space before the other half of the hyphenated word (pater-/ num, regio-/ num). Is this by design? Does this create problems in lemmatization and other text processing operations? If hyphenation does not need to be kept, I can put the hyphenated parts together. If it should be kept, and if there is a standard for encoding such cases, I can mark it up. Let me know.

**Comment by lcerrato** [comment]
No, it's not by design. Seems to be a scanning artifact and appears to be random. (For instance, quemadmodum is hyphenated in the text in the first line, but that's been regularized). 
I'm only guessing at this as there is no quality info in the header.
In P4 this does create morphology problems (regio + num are parsed as separate words).
I would reconstruct the words and not make any effort to preserve the hyphens where they are clearly artifacts of textual end of line breaks.

**Comment by nevenjovanovic** [comment]
OK, have done the changes and will submit a pull request. But how do you deal with hyphenated words at the end of page (before pb element)?

**Comment by lcerrato** [comment]
It's been a while, but I think it used to be that the word goes before the break, be it line or page, etc. Not that there is a wrong answer.

**Comment by nevenjovanovic** [comment]
@lcerrato -- yes, that's how we usually solve this for CroALa too. So I'll connect also the halves separated by page break. I'll do it in another commit for clearer overview.
